.travis.yml

```diff
@@ -21,10 +26,13 @@ before_install:
     - |
       echo ""
       echo "Removing homebrew from Travis CI to avoid conflicts."
+      cp /usr/local/bin/md5deep /tmp || echo "Could not copy md5deep binary, this happens when travis get a cache miss"
```

Comment:
Travis uses ```md5deep``` to detect changes on the cache. But the osx image we use does not have it pre-installed and must get it from homebrew: https://github.com/travis-ci/travis-ci/issues/7456

So before we uninstall homebrew we need to at least save ```md5deep``` binary (there are no dependencies) until the xcode6.4 image is updated or we move to a newer image.